samples/weather_function/readme.md

```diff
@@ -0,0 +1,33 @@
+# Weather Function Sample
+
+This is a Serverless Function that fetches the current weather information from the Open Weather Map [service](https://openweathermap.org/current) via HTTP. Part of the Function's output includes HTTP response headers, and the HTTP response body. This sample is designed to be deployed on OpenFaaS via a Docker image.
+
+
+# Building Docker Image
+
+It will be assumed that Docker is already installed.
+
+1. Clone the Kotlin Native Git repository: `git clone https://github.com/JetBrains/kotlin-native ~/repos/kotlin-native`
+2. Change working directory to the sample: `cd ~/repos/kotlin-native/samples/weather_function`
+3. Create the **openweathermap_key.txt** file and append your [Open Weather Map API key](https://openweathermap.org/appid) to the file
+4. Build the *weather* Docker image by running the shell script: `./build_image`
+
+
+# Usage
+
+Make sure that you have completed the *Building Docker Image* section, and that OpenFaaS is [installed](https://docs.openfaas.com/deployment/) along with the [OpenFaaS CLI](https://github.com/openfaas/faas-cli) before proceeding.
+
+1. Start OpenFaaS
+2. Goto the following to access the OpenFaaS Portal:
+http://127.0.0.1:8080
```

Comment:
This URL is only valid if the user follows the Swarm deployment guide, otherwise it's port `:31112` for Kubernetes (when using a local environment). Might be better to ask them to install on Swarm if you want to assume a quick/happy-path.